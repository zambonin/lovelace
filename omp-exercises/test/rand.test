#!/bin/sh

./omp-exercises/rand_comparing 10
