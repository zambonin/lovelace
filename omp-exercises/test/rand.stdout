a[i] < b[i] 6 times
